utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -85,6 +90,14 @@ public LockClient(UUID clientId, CorfuRuntime corfuRuntime) throws NoSuchMethodE
             return t;
         });
 
+        this.leaseMonitorScheduler = Executors.newScheduledThreadPool(1, (r) ->
+        {
```

Comment:
where do we define what gets executed when this thread runs?

Reply:
You can see that logic in the HasLeaseState class L.196